src/accountsdb/db.zig

```diff
@@ -596,7 +598,7 @@ pub const AccountsDB = struct {
                 // ! reset memory for the next slot
                 defer geyser_storage.reset();
 
-                const data_versioned = sig.geyser.core.VersionedAccountPayload{
+                const data_versioned: sig.geyser.core.VersionedAccountPayload = .{
```

Comment:
It's not important, but just curious why you changed this. Personally I think the old approach was slightly cleaner, but I'm mostly indifferent.

Reply:
I'm basically following the reasoning here https://github.com/ziglang/zig/issues/5038#issuecomment-2080013522. I also just like how it looks tbh :)

Reply:
i prefer the old approach too

Reply:
I am unconvinced by the argument in the linked comment, even for the special case he described. The proposal here would actually make his problem worse, not better.

Even if I agreed with him on that point, the special case seems to be of negligible importance relative to all the other implications of banishing the syntax. I feel there are much stronger arguments in favor of keeping the syntax.

Regardless, I don't think it is consequential enough to block this PR with a debate about it. I'll go ahead and resolve this discussion.